blueocean-core-js/package.json

```diff
@@ -59,8 +57,8 @@
     "chai": "3.5.0",
     "del": "2.2.2",
     "enzyme": "2.4.1",
-    "eslint-plugin-react": "6.3.0",
```

Comment:
This was being ignored because of the entry in "dependencies"; so bringing it to 4.3.0 so it's in alignment.